content/_data/roadmap/roadmap.yml

```
@@ -342,6 +342,25 @@ categories:
       It also includes integrations with cloud storage providers.
     link: /sigs/cloud-native
     initiatives:
+    - name: CloudEvents support
+      status: current
+      description: >
+         Microservices are the de facto architecture of the cloud and events are the primary way in which microservices communicate a change of state.
+         CloudEvents is a specification for describing event data in common, standardised formats.
+         The CloudEVents plugin would enable Jenkins support for CloudEvents by allowing events to be emitted in Jenkins pipelines and triggering Jobs on certain events,
+         bringing Jenkins one step closer to interoperability with other technologies in the cloud.
+      link: https://www.jenkins.io/projects/gsoc/2021/project-ideas/cloudevents-plugin/
+      labels:
+      - feature
+    - name: Tekton Client plugin
+      status: current
+      description: >
+         This plugin enables Tekton as the build engine to carry out all the Jenkins Kubernetes builds.
+         Jenkins should be able to run all its jobs in Tekton and manage them directly through the Jenkins console,
```

I'm assuming that a Tekton job type is distinct from a Pipeline job type, a Freestyle job type, a Maven job type?
```suggestion
         Jenkins should be able to run jobs in Tekton and manage them directly,
```

---

No, It is the Jenkins Jobs that will be converted to run in Tekton.

---

The end goal is that any and all Jenkins Jobs could be run in Tekton.

---

Changed the wording a bit, to make more clear :)